java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/Application.java

```diff
@@ -56,6 +56,20 @@
         .longOpt("server-runtimes-map")
         .desc("Mappings between supported jdk versions, server types, and docker images")
         .build());
+
+    CLI_OPTIONS.addOption(Option.builder("m")
+        .required()
```

Comment:
Can we output a warning and fail later? If this adds too much complexity for what it's worth, I would let it go. I don't feel too strongly about it.

For `EXECUTABLE_NAME`, maybe just `<BUILDER>`?